pages/common/sdcv.md

```diff
@@ -0,0 +1,37 @@
+# sdcv
+
+> StarDict console version. Command line dictionary client.
+> Dictionaries are provided separately.
```

Comment:
Perhaps this will read better?
```suggestion
> StarDict command line dictionary client.
> Dictionaries are provided separately to the program itself.
```

Reply:
Would you like to edit that to `command-line`

I didn't see it previously

And I don't like that `itself`, its unrequired

Reply:
I want to include"StarDict command-line version" somewhere, because that's what the acronym is.

Reply:
How about "StarDict command-line version, dictionary client. Dictionaries are provided separately from the client. "

Reply:
```
> StarDict command-line version (sdcv).
> Dictionaries are provided separately from the program itself.
```
(note: @sbrl's `separately to` is not grammatically correct — @258204's `separately from` is)

Reply:
Hrm, `StarDict command-line version` doesn't really read very well or make grammatical sense. Despite this, I think @bl-ue's suggestion is the best one given the situation.

Reply:
What about 

```
> StarDict command-line variant
```

Reply:
Hrm. I think `StarDict command-line variant` is worse than @bl-ue's version.